### DISCUSSION TITLE
Implement stacking to a universal eVscope camera?

### DISCUSSION THREAD

**Discussion by astrotater** [artyom-beilis/OpenLiveStacker#93] [General]
I’m working on a Raspberry Pi Zero 2W, Raspberry Pi HQ Camera, and 1.28 inch 240x240 round LCD based eVscope type image intensifier. 
If I can get it to work, my goal is a <$100 eyepiece that can turn any telescope into a type of Unistellar eVscope. 

Would it be possible to implement the live stacking and alt/az field derotation into this camera? Since it’s a Pi Zero 2W, it might have to use OpenCV lite. 
I’m thinking something along the lines of implementing the live stack that intensifies the image seen on the 1.28” TFT “eyepiece” and improves it over time, just like the eVscope and SeeStar S50 do. A “live view” mode for centering, then once centered, stack the image for as long as the user would like. 
I don’t think plate solving or any other features would be necessary. Just basic image intensification. 

![IMG_0247](https://github.com/artyom-beilis/OpenLiveStacker/assets/171597037/afe14f63-ae5c-4f68-96ff-a329ef77c465)
![IMG_0246](https://github.com/artyom-beilis/OpenLiveStacker/assets/171597037/62598453-bb75-430b-8e38-cd2021358f21)

**Comment by artyom-beilis** [comment]
I don't see why not.

Regarding PI. It seems to have good 64 bit CPU but a small RAM of 512MB which is serious limit - since you need memory for OS and other goodies. It may be problematic for stacking large images, but probably would work in low frame rate. I can't tell for sure.

However in order to make it useful you need a large sensor. If you are talking about IMX477 it is tiny. Even with 400mm FL you'll have a very small ~0.67 degree  FOV. So you'll likely need goto and tracking. Good optics to show the display.
